packages/button/package.json

```diff
@@ -31,6 +25,8 @@
   },
   "homepage": "https://github.com/CraveFood/farmblocks#readme",
   "dependencies": {
+    "@crave/farmblocks-constants": "^1.1.0",
```

Comment:
this can be removed now

Reply:
I removed on https://github.com/CraveFood/farmblocks/pull/94.
I'll merge and rebase into this one

Reply:
done